lib/generators/templates/wor_paginate.rb

```diff
@@ -22,4 +22,11 @@
   # Wor::Paginate::Adapters::Kaminari
   # Wor::Paginate::Adapters::ActiveRecord
   # Wor::Paginate::Adapters::Enumerable
+
+  # In case you want to use other adapter for your response, you can override our adapter here
+  # You can extend from Wor::Paginate::Adapters::Base and override the methods 'required_methods',
+  # 'paginated_content', 'total_pages', 'total_count'
+  # Then you'll simply have to call render_paginated(content, adapter: CustomAdapter)
+  # For more details on how to implement a custom adapter, or make operations in the gem's adapters
+  # take a look at the README in the 'Custom adapters' section: https://github.com/Wolox/wor-paginate#custom-adapters
```

Comment:
:thinking: It's ok. Keep it in this way.

Personally, I prefer to keep this template more concise as possible. If I need more information about this configuration, I check out the Readme or the wikis gem pages

```rb
  # Custom adapters
  #
  # config.adapters = [Adapters::KaminariAlreadyPaginated, Adapters::WillPaginateAlreadyPaginated, Adapters::WillPaginate, Adapters::Kaminari, Adapters::ActiveRecord, Adapters::Enumerable]
  # config.add_adapter(adapter)
  # config.remove_adapter(adapter)
  # config.clear_adapters
  # config.reset_adapters!
```

Reply:
I tried to maintain the same explanation text that is above for custom formatters `In case you want to use other format for your response, you can override our formatter here...`. Is no problem to delete this explanation from here, but if you want to be consistent, that custom formatters explanation should be also removed from here.